ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/rewrite/testtool/tests/Time.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.rewrite.testtool.tests
+
+import java.time.Duration
+
+import com.daml.ledger.api.rewrite.testtool.infrastructure.{
+  LedgerSession,
+  LedgerTest,
+  LedgerTestSuite
+}
+
+import scala.concurrent.duration.DurationInt
+
+final class Time(session: LedgerSession) extends LedgerTestSuite(session) {
+
+  val pass =
+    LedgerTest("Advancing time should return the new time") { implicit context =>
```

Comment:
Is this test missing a `skipIf` clause for when the time service is not available?

Reply:
Kinda. The problem is that I haven't found a way to express it reliably. Basically right now any option I could introduce would more ore less be equivalent to explicitly `--include=Time` from the CLI.

Reply:
Superseded by https://github.com/digital-asset/daml/pull/2282#discussion_r311404661